pkg/generators/openapi.go

```diff
@@ -328,6 +328,7 @@ func (g openAPITypeWriter) generateMembers(t *types.Type, required []string) ([]
 			required = append(required, name)
 		}
 		if err = g.generateProperty(&m, t); err != nil {
+			glog.Errorf("Error when generating: %v, %v\n", name, m)
```

Comment:
Returned error doesn't contain information on for what it failed. We can add it there, but this was just simpler.